lib/mcb/commands/apiv1/find.rb

```diff
@@ -0,0 +1,25 @@
+name 'find'
```

Comment:
I can't seem to get this one to work. What argument does it take? Provider code?

Reply:
It assumes you have `manage-courses-backend` running locally on port 3001, and a db populated with providers. If you had that setup, what was your error?

Reply:
I got the error in the error message before, now that you have fixed it I get:

```
❯ bin/mcb apiv1 find A1
looking for provider A1
Provider with code 'A1' not found
```

Reply:
Do you have a provider with the code `A1` in your DB? All the provider codes in the Real Data (tm) have 3 characters, did you add one with 'A1'?